server/template.go

```diff
@@ -153,17 +153,17 @@ func init() {
 `))
 
 	template.Must(masterTemplate.New("commentMentionNotification").Funcs(funcMap).Parse(`
-{{template "user" .GetSender}} mentioned you on [{{.GetRepo.GetFullName}}#{{.Issue.GetNumber}}]({{.GetComment.GetHTMLURL}}):
+{{template "user" .GetSender}} mentioned you on [#{{.Issue.GetNumber}} {{.Issue.GetTitle}}]({{.GetComment.GetHTMLURL}}):
```

Comment:
@iwataka Do you think it makes sense to keep the repo name in the message? I think it provides value so the dev knows which repo the PR/issue is in upon inspection.

Reply:
I agree that the repo name is useful. Should we do `Name` instead of `Full Name` so it does not get too wordy with the title? 

Something like: 

[mattermost-plugin-github#136](https://github.com/mattermost/mattermost-plugin-github/pull/136) - Add PR and Issue Title in Notification DMs

Thoughts @mickmister

Reply:
@marianunez Sounds good to me. I'm trying to think of non-MM org users, and if they frequently have interactions with repos from different orgs. Looking at my DM feed, it certainly seems redundant to have the org show on each message.

:+1: On just including repo name without the org

Reply:
@mickmister Thanks for your suggestion and it makes sense.
So I will change template strings like the below:
> mattermost-plugin-github#136 - Add PR and Issue Title in Notification DMs

Reply:
Finished @mickmister

Reply:
@iwataka FYI, our convention for PRs is to merge instead of rebase, so we can easily tell the differences from the last time we reviewed. No worries though, just a heads up